styles/layouts/_main-navigation.scss

```diff
@@ -311,6 +311,11 @@
     height: 12px;
   }
   .navbar-explore {
+
+    .dropdown-menu {
+      max-width: 22.563rem;
```

Comment:
pretty random number

Reply:
its 361 px to rem , matched figma dimensions :D

Reply:
Does it look good on small screen? Do we need to set a max-width?

Reply:
@prachi00 important is to keep the fixed padding, like 24px etc, not fixed container width that may change

Reply:
cc @prachi00